src/components/box/box.component.js

```diff
@@ -9,6 +10,40 @@ const Box = styled.div`
   ${layout}
   ${flexbox}
   ${color}
+
+  ${({ scrollVariant, theme }) => css`
+    ${scrollVariant === "light" &&
+    css`
+      scrollbar-color: ${theme.scrollbar.light.thumb}
```

Comment:
Theres a lot of duplication between the light and dark variants. We could simplify this by using the prop like this
```suggestion
      scrollbar-color: ${theme.scrollbar[scrollVariant]thumb}
```

Reply:
This is definitely more efficient